modules/imgproc/src/subdivision2d.cpp

```diff
@@ -506,9 +597,9 @@ void Subdiv2D::initDelaunay( Rect rect )
     topLeft = Point2f( rx, ry );
     bottomRight = Point2f( rx + rect.width, ry + rect.height );
 
-    Point2f ppA( rx + big_coord, ry );
-    Point2f ppB( rx, ry + big_coord );
-    Point2f ppC( rx - big_coord, ry - big_coord );
+    Point2f ppA( 1.f, 0.f );
```

Comment:
the initialization has been changed; is this particular change important to fix the bug? I mean, it will result in a slightly different triangulation. Of course, the meaningful part of the triangulation will stay the same, but maybe some software relies on this particular initialization.